content/docs/command-reference/dag.md

```diff
@@ -1,49 +1,49 @@
 # dag
 
-Show [stages](/doc/command-reference/run) in a pipeline that lead to the
-specified stage. By default it lists
-[DVC-files](/doc/user-guide/dvc-files-and-directories).
+Display connected [stages](/doc/command-reference/run) in the pipelines that
```

Comment:
Visualize pipelines from the `dvc.yaml` file as a graph of connected stages.

Reply:
Done